node/tessel-export.js

```diff
@@ -916,6 +916,10 @@ Tessel.LED.prototype.toggle = function(callback) {
   this.write(this.value ? 0 : 1, callback);
 };
 
+Tessel.LED.prototype.isOn = function() {
+  return this.value === 1;
+};
+
```

Comment:
> Don't know if you saw this since I edited, but what do you think?

I did, I don't care for this change at all. "on" and "off" are behaviors and object behaviors are represented by methods of the object. This is the kind of design that gets papered over by user code: 

```js
function on() {
  led.on = true;
}

function off() {
  led.off = true;
}
```

-----------

In all honesty, Johnny-Five's `Led` class produces instances that have `on()`, `off()`, `toggle()`, `isOn` and in 3 years I haven't had any requests to make `on` and `off` into properties or `isOn` into a method. In real use cases, the design [looks and works quite nicely](http://johnny-five.io/examples/proximity-EVS_EV3_US-alert/). I will be personally burdened with the task of explaining to developers that are making LED things with Johnny-Five (on a Tessel) why they'd have to write this: 

```js
var white = new five.Led("a7");
white.on();

var green = Tessel.leds[2];
green.on = true;
```

Instead of: 

```js
var white = new five.Led("a7");
white.on();

var green = Tessel.leds[2];
green.on();
```


And

```js
if (white.isOn) { ... }

if (green.isOn()) { ... }
```

Instead of: 

```js
if (white.isOn) { ... }

if (green.isOn) { ... }
```